ext/standard/tests/strings/gh19898.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+GH-19899 (undefined behavior with zend_strtod with large digits as strings)
+--INI--
+memory_limit=-1
+--SKIPIF--
+<?php if (PHP_INT_SIZE != 8) die("skip this test is for 64bit platform only"); ?>
+--FILE--
+<?php
+str_repeat("1", (int)str_repeat("1", 2147483648));
```

Comment:
Question: do you need the _nested_ invocation? Intuitively I thought that the cast is the one invoking zend_strtod() ?